schema/spacewalk/common/tables/accessGroup.sql

```diff
@@ -20,5 +20,5 @@ CREATE TABLE access.accessGroup (
 );
 COMMENT ON TABLE access.accessGroup IS 'Access groups (roles) for RBAC';
 
-CREATE UNIQUE INDEX access_group_label_uq
-ON access.accessGroup(label);
+CREATE UNIQUE INDEX access_group_org_label_uq
+ON access.accessGroup(org_id, label);
```

Comment:
If I remember correctly the org_id can be null. How will this unique index will behave? 
It will allow to have the same label at different organizations, but will those names collide with the default group labels?
For example, if we have a default group name "imageAdmin", with org_id null, can the user create new group with the same label assigned to an org?
I think we must prevent this.

Reply:
I can do it with an extra check constraint on reserved groups.

Reply:
Fixed in `AccessGroupManager`